tests/framework/ensembleModelTests/gold/ensembleModelFullyCorrVars/printFinalResults.csv

```diff
@@ -1,7 +1,7 @@
 sigma-A,sigma-B,decay-A,decay-B,A,B,C,D,sum
-224.664963136,224.664963136,6.70983045425e-08,6.70983045425e-08,0.0452587382226,0.233491287235,0.877803666103,1.56213572566,2.71868941722
```

Comment:
is it the tol <=1e-8 that you needed to regold?

Reply:
This is because Python 3 gives different answers than Python 2, but I am not sure why.